src/stylesheets/components/_header.scss

```diff
@@ -0,0 +1,602 @@
+// Navigation ------------- //
+
+$height-nav-secondary: 5rem;
+$width-nav-sidebar:    250px;
+
+
+.usa-header {
+  width: 100%;
+  z-index: 1;
+
+  @include media($nav-width) {
+    border-bottom: 1px solid $color-gray-light;
+  }
+
+  a {
+    border-bottom: none;
+  }
+
+  em {
```

Comment:
I find that confusing. As a developer, I don't think I'd understand that the `em` is supposed to be a logo.